tests/Api/Request/ParametersBuilderTest.php

```diff
@@ -230,7 +230,32 @@ public function setSearchParamsProvider()
                         'cat_id' => [125]
                     ]
                 ]
-            ]
+            ],
+            'Request with same attributes' => [
+                'parameters' => [
+                    Plugin::API_PARAMETER_ATTRIBUTES => [
+                        'vendor' => [
+                            'A & C Design',
+                            'A & C Design',
+                        ]
+                    ],
+                ],
+                'requestUri' => 'https://www.test.com/search?query=Test',
+                'category' => false,
+                'expectedParameters' => [
+                    'query' => 'Test',
+                    'properties' => [
+                        0 => 'variation_id'
+                    ],
+                    'attrib' => [
+                        'vendor' => [
+                            'A & C Design',
+                        ]
+                    ],
+                    'order' => 'price ASC',
+                    'count' => 10
+                ]
+            ],
```

Comment:
Can you please also test any logic related to min and max.

Reply:
There is already a test that tests for same min and max value. I now added a test that checks both min and max is persisted but same filter values are removed.